CWE/history/countries/ZAN - Zanzibar.txt

```diff
@@ -2,15 +2,15 @@ capital = 2048
 primary_culture = swahili
 culture = arab
 religion = secularism
-government = hms_government
+government = absolute_monarchy
```

Comment:
Reforms need to be edited to prevent auto switch to hms gov

Reply:
Which one in particular, I thought none_voting was enough

Reply:
You can just use the cleanup events found in political_revolution.txt, there are events for each set of reforms I believe.